roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,173 @@
+"""
+Module for performing checks on an Fluentd logging deployment configuration
+"""
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+import json
+import os
+
+
+class FluentdConfig(OpenShiftCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    logging_namespace = None
+
+    @classmethod
+    def is_active(cls, task_vars):
+        return super(FluentdConfig, cls).is_active(task_vars)
+
+    def run(self, tmp, task_vars):
+        """Check that Fluentd has running pods, and that its logging config matches Docker's logging config."""
+
+        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
+        fluentd_pods, error = self.get_pods_for_component(
```

Comment:
I was hesitant to rely on the logging var since its value is not set by default, but I guess for now it might be the easiest starting point for this. Will update